### PR TITLE
[16.0][MIG] sale_partner_company_group

### PR DESCRIPTION
**Pull request by Highcooley**

### CODE REVIEW

**Review by stefan-tecnativa — APPROVED** [review]
Tested. LGTM 

TT44356

**Comment by km-landoo** [comment]
@Highcooley could you reload the runbot pls?

**Comment by stefan-tecnativa** [comment]
You should squash your last commit into the migration one. Can you do it please?

**Comment by Highcooley** [comment]
@stefan-tecnativa Done

File: sale_partner_company_group/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Sale Partner Company Group",
     "summary": "Adds the possibility to add a company group to a company",
-    "version": "15.0.1.0.1",
+    "version": "16.0.0.1.0",
```
Comment (pedrobaeza) [inline sale_partner_company_group/__manifest__.py]: Incorrect version: It should be `16.0.1.0.0`

File: sale_partner_company_group/i18n/partner_company_group.pot
Comment (pedrobaeza) [inline sale_partner_company_group/i18n/partner_company_group.pot:1]: This file should go out

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
I still see 2 migration commits, and you can squash commits this way:

Selección_034

**Comment by Highcooley** [comment]
@pedrobaeza partner_company_group.pot removed, however the manifest is already on 16.0.1.0.0

**Comment by pedrobaeza** [comment]
OK, the change was done in other commit. What about the squashing of the other commits?

**Comment by Highcooley** [comment]
@pedrobaeza here we go

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**